relay-server/src/utils/rate_limits.rs

```diff
@@ -515,7 +523,11 @@ impl Enforcement {
             ItemType::Span | ItemType::OtelSpan | ItemType::OtelTracesData => {
                 !self.spans_indexed.is_active()
             }
-            ItemType::ProfileChunk => !self.profile_chunks.is_active(),
+            ItemType::ProfileChunk => match item.profile_type() {
+                Some(ProfileType::Backend) => !self.profile_chunks.is_active(),
+                Some(ProfileType::Ui) => !self.profile_chunks_ui.is_active(),
+                None => true,
```

Comment:
Will this reject or accept profiles when we don't have a type?

Reply:
It will accept when there is no type.